mpu6050_node:
  ros__parameters:
    i2c_bus: 7
    i2c_address: 104  # 0x68
    rate_hz: 50.0
    accel_bias: [0.0, 0.0, 0.0]
    gyro_bias: [0.0, 0.0, 0.0]

imu_punch_classifier:
  ros__parameters:
    enable_punch_classification: true
    window_size: 10
    cooldown_s: 0.5
    gyro_threshold: 2.5
    accel_threshold: 6.0
    accel_peak_ratio: 2.0
    gyro_peak_ratio: 2.0
    axis_dominance_ratio: 1.3
    imu_hand: right
    calibration_path: "~/.boxbunny/imu_calibration.json"
    use_calibration: true
